include/bluetooth/mesh/gen_lvl_srv.h

```diff
@@ -64,6 +64,12 @@ struct bt_mesh_lvl_srv_handlers {
 			  struct bt_mesh_lvl_status *rsp);
 
 	/** @brief Set the level state.
+	 *
+	 * When a set message is received, the model publishes a status message, with the response
+	 * set to @c rsp. When an acknowledged set message is received, the model also sends a
+	 * response back to a client. If a change of the state is not instantaneous, e.g. when
+	 * @ref bt_mesh_model_transition_time returns a nonzero value, the application is
+	 * responsible for publishing a value of the level state at the end of transition.
```

Comment:
Other states you wrote from the capital letter but not the level

Reply:
I followed the pattern in this file. Level is not capitalized here.